SU2_CFD/include/fluid/CConstantSchmidtRANS.hpp

```diff
@@ -0,0 +1,65 @@
+/*!
+ * \file CConstantSchmidtRANS.hpp
+ * \brief Defines a mass diffusivity model with constant Schmidt numbers for RANS.
+ * \author S. Vitale, M. Pini, G. Gori, A. Guardone, P. Colonna, T. Economon
+ * \version 7.4.0 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "CDiffusivityModel.hpp"
+
+/*!
+ * \class CDiffusivityModelRANS
+ * \brief Defines a mass diffusivity model for species equations based on Schmidt numbers for RANS.
+ * \author T. Economon
+ */
+class CConstantSchmidtRANS : public CDiffusivityModel {
+ protected:
+  // su2double Schmidt_Lam;   /*!< \brief Laminar Schmidt number. */
+  // su2double Schmidt_Turb;  /*!< \brief Laminar Schmidt number. */
```

Comment:
Clean up the commented out sections please.

Reply:
Ok, I will do it. Thank you so much for the comments.